manifests/server.pp

```diff
@@ -328,7 +328,7 @@
   Optional[Stdlib::Absolutepath] $pidfilepath                             = undef,
   String[4,4] $pidfilemode                                                = '0644',
   Boolean $manage_pidfile                                                 = false,
-  String $rcfile                                                          = "${facts['root_home']}/.mongorc.js",
+  String $rcfile                                                          = "${facts['root_home']}/.mongoshrc.js",
```

Comment:
I really don't like this. This only affects where the rc file gets stored, but not from where it is read, that is hardcoded in the provider. See this same commit, lib/puppet/provider/mongodb.rb, line 15.